src/elliptic/curves/secp256_k1.rs

```diff
@@ -32,7 +32,7 @@ use crate::arithmetic::*;
 use super::traits::*;
 
 lazy_static::lazy_static! {
-    static ref CONTEXT: secp256k1::Secp256k1<secp256k1::VerifyOnly> = secp256k1::Secp256k1::verification_only();
+    static ref CONTEXT: secp256k1::Secp256k1<secp256k1::All> = secp256k1::Secp256k1::new();
```

Comment:
@elichai, is it fine that I turned on all capabilities? I had to do this because this is how PublicKey can be constructed from SecretKey (generator multiplication basically)

Reply:
Yeah, that's the right thing to do, it alllows multiplying by the generator